### PR TITLE
Update TPC QC Trending

### PR DESCRIPTION
**Pull request by mlesch**

### CODE REVIEW

File: Modules/TPC/src/TrendingTaskTPC.cxx [hunk 1]
```diff
@@ -187,81 +260,166 @@ void TrendingTaskTPC::drawCanvas(TCanvas* thisCanvas, const std::string& var,
...
+    for (int i = 0; i < xBins; i++) {
+      xBoundaries[i] = axis[0][i];
+    }
+    const int yBins = axis[1].size();
+    float yBoundaries[yBins];
+    for (int i = 0; i < yBins; i++) {
+      yBoundaries[i] = axis[1][i];
+    }
+
+    TH2F* graph2D = new TH2F("Graph2D", "", xBins - 1, xBoundaries, yBins - 1, yBoundaries);
```
Comment (wiechula) [inline Modules/TPC/src/TrendingTaskTPC.cxx:389]: Can several histograms with the same name exist in on e session? If so, root will delete the previous one.

**Review by wiechula — CHANGES_REQUESTED** [review]

File: Modules/TPC/src/TrendingTaskTPC.cxx (hunk 1 above)
Comment (mlesch) [reply Modules/TPC/src/TrendingTaskTPC.cxx:364]: Normally this works as the TH2F is will be associated to a primitive of the canvas, but I will check this in more detail for the next bigger update

**Review by wiechula — APPROVED** [review]

File: Modules/TPC/src/TH1ReductorTPC.cxx [hunk 2]
```diff
@@ -115,51 +128,49 @@ void TH1ReductorTPC::update(TObject* obj, std::vector<SliceInfo>& reducedSource,
...
-    StddevY += pow(MeanY - Hist->GetBinContent(i), 2.);
+  for (int i = lowerBin; i <= upperBin; i++) {
+    stddevY += pow(meanY - hist->GetBinContent(i), 2.);
   }
 
-  StddevY /= ((float)IterateBins - 1.);
-  ErrMeanY = StddevY / ((float)IterateBins);
-  StddevY = sqrt(StddevY);
-  ErrMeanY = sqrt(ErrMeanY);
+  stddevY /= ((float)iterateBins - 1.);
```
[the remark this answers is not in the document]
Comment (mlesch) [reply Modules/TPC/src/TH1ReductorTPC.cxx:166]: I do not think that we need an explicit protection. A few lines above, there is 
`const int iterateBins = upperBin - lowerBin + 1;`
and 
```
if (upperBin <= lowerBin) {
    ILOG(Error, Support) << "Error: Upper bin smaller than lower bin in TH1ReducterTPC::GetTH1StatsY" << ENDM;
    exit(0);
  }
```

So `iterateBins` is always at least 2.

[the remark this answers is not in the document]
Comment (wiechula) [reply Modules/TPC/src/TH1ReductorTPC.cxx:166]: Very good!

**Pull request merged**